integration/src/test/java/org/mule/test/core/context/notification/PipelineMessageNotificationLogger.java

```diff
@@ -8,24 +8,14 @@
 
 import org.mule.runtime.api.notification.PipelineMessageNotificationListener;
 import org.mule.runtime.api.notification.PipelineMessageNotification;
+import org.mule.tck.core.context.notification.AbstractNotificationLogger;
 
-import java.util.List;
-
-public class PipelineMessageNotificationLogger extends PipelineAndAsyncMessageNotificationLogger
+public class PipelineMessageNotificationLogger extends AbstractNotificationLogger<PipelineMessageNotification>
```

Comment:
make this change also in `AsyncMessageNotificationLogger` and remove `PipelineAndAsyncMessageNotificationLogger `